contrib/kernel-install/91-sbctl.install

```diff
@@ -27,6 +27,14 @@ case "$COMMAND" in
 add)
 	[ "$KERNEL_INSTALL_VERBOSE" -gt 0 ] &&
 		printf 'Signing kernel %s\n' "$IMAGE_FILE"
+
+	# exit without error if keys don't exist
+    # https://github.com/Foxboron/sbctl/issues/187
+	if ! test -d /usr/share/secureboot; then
```

Comment:
`sbctl --json status | grep installed | awk '{print $2}' | tr -d ','`

or something like this would probably be a better idea?

Reply:
Seems like that doesn't always output something when the keys exist?

```
# sbctl create-keys
Created Owner UUID e8be777f-2cc7-4da5-bcfa-36fd80241bca
Creating secure boot keys...✓
Secure boot keys created!
c1ef6e49abfa / # sbctl --json status | grep installed | awk '{print $2}' | tr -d ','
c1ef6e49abfa / # sbctl --json status
```

Reply:
Uhhhhhh, that is a bug :)

Reply:
I can reproduce like:

```
$ docker run --volume ~/git/sbctl:/sbctl -it archlinux bash
[root@f5c510c0e4a7 sbctl]# pacman --noconfirm --noprogressbar -Syu
[snip]
[root@f5c510c0e4a7 sbctl]# pacman --noconfirm --noprogressbar -S make go asciidoc gcc git
[snip]
[root@f5c510c0e4a7 sbctl]# go build -buildvcs=false -o sbctl ./cmd/sbctl
[snip]
[root@f5c510c0e4a7 sbctl]# ./sbctl create-keys
Created Owner UUID e0812b04-a6dd-4344-99a1-4691f4c20e07
Creating secure boot keys...✓
Secure boot keys created!
[root@f5c510c0e4a7 sbctl]# ./sbctl --json status
```

.. where ~/git/sbctl is this repository, updated to dcdc703, latest at the time of writing

Reply:
Right, the docker container has access to efivars? Alternatively, do you get anything dropping `--json`?

Reply:
Ah, yes:

```
 ./sbctl status
system is not booted with UEFI
```

Which makes sense, because the container is indeed not booted with UEFI. So in this way, `sbctl status` seems unsuitable as a condition for signing, because signing *does* work after only doing `sbctl create-keys` in this environment.

Reply:
I agree, thanks!